cardano-sync/src/Cardano/Sync/LedgerState.hs

```diff
@@ -222,6 +216,15 @@ initCardanoLedgerState pInfo = CardanoLedgerState
       { clsState = Consensus.pInfoInitLedger pInfo
       }
 
+-- TODO make this type safe. We make the assumption here that the first message of
+-- the chainsync protocol is 'RollbackTo'.
+readStateUnsafe :: LedgerEnv -> STM CardanoLedgerState
+readStateUnsafe env = do
+    mState <- readTVar $ leStateVar env
+    case mState of
+      Nothing -> panic "ledger state is not found"
```

Comment:
I realize this is still a work in progress, but `panic` message should have the module name in the message so if it ever gets hit we know where it came from.